samples/drivers/touch/src/main.c

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2024 Antmicro <www.antmicro.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/logging/log.h>
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/input/input.h>
+#include <zephyr/drivers/display.h>
+
+LOG_MODULE_REGISTER(sample, LOG_LEVEL_INF);
+
+#define BUFFER_SIZE  (CONFIG_CROSS_WIDTH * CONFIG_CROSS_HEIGHT * 4)
```

Comment:
As this can amount to a non-negligible amount of memory (1.6 KB) even on RGB565, monochrome, etc displays, it might be better to allocate based on bpp?

Reply:
Switched to using BPP here